typescript/infra/scripts/funding/fund-keys-from-deployer.ts

```diff
@@ -65,29 +65,29 @@ const nativeBridges = {
   },
 };
 
-type L2Chain =
-  | Chains.optimism
-  | Chains.optimismgoerli
-  | Chains.arbitrum
-  | Chains.arbitrumgoerli
-  | Chains.base;
-
-const L2Chains: ChainName[] = [
-  Chains.optimism,
-  Chains.optimismgoerli,
+type L2Chain = (typeof L2Chains)[number];
```

Comment:
it's better to divide the l2 as OpStack, ArbNitro, etc which all have the same funding mechanism (contract interaction) and then union them to get L2Chains

Reply:
good idea